### PR TITLE
sync the handshaker.proto with the internal version

### PR DESCRIPTION
**Pull request by rockspore**

### CODE REVIEW

File: grpc/gcp/handshaker.proto [hunk 1]
```diff
@@ -125,11 +125,11 @@ message ServerHandshakeParameters {
   // specified. Otherwise, the handshaker chooses a default local identity.
   repeated Identity local_identities = 2;
 
-  // (Optional) An access token created by the caller only intended for use in
-  // ALTS connections. The access token that should be used to authenticate to
-  // the peer. The access token MUST be strongly bound to the ALTS credentials
-  // used to establish the connection that the token is sent over.
-  optional string access_token = 3;
+  // An identity token created by GKE MDS only intended for use in Direct Path
```
Comment (matthewstevenson88) [inline grpc/gcp/handshaker.proto:128]: Can we leave the original wording as-is and just omit the word "access"?

**Review by matthewstevenson88 — COMMENTED** [review]
Thanks for the PR Luwei!

File: grpc/gcp/handshaker.proto (hunk 1 above)
Comment (rockspore) [reply grpc/gcp/handshaker.proto]: Makes sense. Done.

File: grpc/gcp/handshaker.proto [hunk 2]
```diff
@@ -125,11 +125,11 @@ message ServerHandshakeParameters {
   // specified. Otherwise, the handshaker chooses a default local identity.
   repeated Identity local_identities = 2;
 
-  // (Optional) An access token created by the caller only intended for use in
-  // ALTS connections. The access token that should be used to authenticate to
-  // the peer. The access token MUST be strongly bound to the ALTS credentials
+  // An token created by the caller only intended for use in
```
Comment (matthewstevenson88) [inline grpc/gcp/handshaker.proto:128]: nit: s/An/A

**Review by matthewstevenson88 — APPROVED** [review]

File: grpc/gcp/handshaker.proto (hunk 2 above)
Comment (rockspore) [reply grpc/gcp/handshaker.proto]: Oops. Thanks!

**Pull request merged**